src/SFA.DAS.ApplyService.Data/AssessorRepository.cs

```diff
@@ -229,11 +230,12 @@ public async Task<int> GetApplicationsInModerationCount()
                     .ExecuteScalarAsync<int>(
                         $@"SELECT COUNT(1)
 	                      FROM Apply apply
-	                      WHERE apply.DeletedAt IS NULL AND (Assessor1ReviewStatus = @approvedReviewStatus AND Assessor2ReviewStatus = @approvedReviewStatus) AND ISNULL(ModerationStatus, 'New') <> @completedModerationStatus",
+	                      WHERE apply.DeletedAt IS NULL AND (Assessor1ReviewStatus = @approvedReviewStatus AND Assessor2ReviewStatus = @approvedReviewStatus) AND ISNULL(ModerationStatus, 'New')  IN (@newModerationStatus, @inProgressModerationStatus)",
                         new
                         {
                             approvedReviewStatus = AssessorReviewStatus.Approved,
-                            completedModerationStatus = ModerationStatus.Complete
+                            newModerationStatus = ModerationStatus.New,
+                            ModerationStatusInProgress = ModerationStatus.InProgress
```

Comment:
typo here - should be:
**inProgressModerationStatus  = ModerationStatus.InProgress**